example/example.c

```diff
@@ -209,6 +210,7 @@ int main(int argc, char* argv[])
 	PDI_expose("dsize", dsize, PDI_OUT);
 	PDI_expose("psize", psize, PDI_OUT);
 	PDI_expose("pcoord", pcoord, PDI_OUT);
+	PDI_event("init");
```

Comment:
why not use event "main_loop" instead in Deisa. Moving the event might have some unexpected side issues

Reply:
you are right. I can use `main_loop`.